scripts/install.sh

```diff
@@ -403,7 +403,7 @@ function get_versions() {
     | sed 'N;N;s/.*true.*//' \
     | grep -o 'v.*"' \
     | sort -r \
-    | sed 's/^v//;s/"$//'
+    | sed 's/^v//;s/"$//' || echo ""
```

Comment:
wouldn't `|| true` also work?
https://stackoverflow.com/a/36131305

Reply:
not entirely sure if it will work for all environments. We have mixed `true` and `echo ""` across the script so decided to use `echo` way.

Reply:
> not entirely sure if it will work for all environments

I love bash.
If we already have that mixed then ok, let's leave it, but I think we should make a standard for it at some point